### PR TITLE
Fixed Dropdown issues

### PR DESCRIPTION
**Pull request by igorkorsukov**
https://www.dropbox.com/s/3m9f362rapy6f5j/Dropdown%20Box%20v1.pdf?dl=0

### CODE REVIEW

File: src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    property int popupItemsCount: root.defaultPopupItemsCount()
+
+    property alias navigation: navCtrl
+
+    property alias dropIcon: dropIconItem
+
+    property string textRole: "text"
+    property string valueRole: "value"
+
+    text: root.displayText
```
Comment (vpereverzev) [inline src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml:53]: Is there any particular reason to have 3 properties like "displayedText", "currentText" and "text" simultaneously? It's a bit confusing which one to use

File: src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+            return root.model[index][roleName]
+        }
+
+        // Complex models (like QAbstractItemModel) with multiple role names
+        if (!(index < delegateModel.count)) {
+            return def
+        }
+
+        var item = delegateModel.items.get(index)
```
Comment (vpereverzev) [inline src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml:95]: maybe we can get it directly from the root.model or even from the ListView.model

File: src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml (hunk 1 above)
Comment (igorkorsukov) [reply src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml:53]: hmm, I will remove displayedText

File: src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml (hunk 2 above)
Comment (igorkorsukov) [reply src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml:95]: unfortunately no, but I think maybe we need our own class for accessing to QAbstractListModel in Qml

Comment (igorkorsukov) [reply src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml:95]: I will do it differently in the next PR

**Review by vpereverzev — APPROVED** [review]

**Pull request merged**